### PR TITLE
Check is organization mgt enabled

### PR DESCRIPTION
**Pull request by AnuradhaSK**
### Proposed changes in this pull request
$Subject

depends on
- [x] https://github.com/wso2-extensions/identity-organization-management/pull/114
- [x] Bump Organization mgt dependency version

### CODE REVIEW

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceDataHolder.java [hunk 1]
```diff
@@ -607,4 +609,27 @@ public void setAdaptiveAuthenticationAvailable(boolean adaptiveAuthenticationAva
 
         isAdaptiveAuthenticationAvailable = adaptiveAuthenticationAvailable;
     }
+
+    /**
+     * Get is organization management enabled.
+     *
+     * @return True if organization management is enabled.
+     */
+    public boolean isOrganizationManagementEnable() {
```
Comment (mefarazath) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceDataHolder.java:618]:
```suggestion
    public boolean isOrganizationManagementEnabled() {
```

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java [hunk 2]
```diff
@@ -2429,6 +2429,16 @@ public static boolean isAdaptiveAuthenticationAvailable() {
         return FrameworkServiceDataHolder.getInstance().isAdaptiveAuthenticationAvailable();
     }
 
+    /**
+     * This method is to check whether organization management is enabled.
+     *
+     * @return Organization management feature is enabled or not.
+     */
+    public static boolean isOrganizationManagementEnable() {
```
Comment (mefarazath) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java:2437]:
```suggestion
    public static boolean isOrganizationManagementEnabled() {
```

Comment (AnuradhaSK) [reply components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java]: fixed in [077830a](https://github.com/wso2/carbon-identity-framework/pull/4165/commits/077830a9e7ab93ed3531984fe2f9c7ce154b8ab2)

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceDataHolder.java (hunk 1 above)
Comment (AnuradhaSK) [reply components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceDataHolder.java]: fixed in [077830a](https://github.com/wso2/carbon-identity-framework/pull/4165/commits/077830a9e7ab93ed3531984fe2f9c7ce154b8ab2)

**Review by dewniMW — APPROVED** [review]

**Pull request merged**